test/functional/apps/settings/_index_pattern_popularity.js

```diff
@@ -48,9 +47,11 @@ define(function (require) {
             return common.sleep(1000);
           })
           .then(function openControlsByName() {
-            return settingsPage.openControlsByName(fieldName);
+            common.debug('Starting openControlsByName "geo.coordinates"');
+            return settingsPage.openControlsByName('geo.coordinates');
```

Comment:
Can the `fieldName` variable be ditched now since you're hardcoding the field name?